app/services/generate_bill_run.service.js

```diff
@@ -92,11 +99,18 @@ class GenerateBillRunService {
     return update
   }
 
+  static _invoicePatch (minimumChargePatch) {
+    return {
+      ...minimumChargePatch,
+      minimumChargeInvoice: true
+    }
+  }
+
```

Comment:
I'm guessing this was leftover from the original attempt to implement this change? But from what I can see I don't think it's needed any longer. I could be wrong! 🤷 

```suggestion
```